releasenotes/notes/expvars-dogstatsd-listeners-5b4aefe7f494964b.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    [dogstatsd] expose number of packets received for each listeners through expvar
```

Comment:
This might look a bit weird in the [changelog](https://github.com/DataDog/datadog-agent/blob/master/CHANGELOG.rst) since we don't usually use `[*]`

Reply:
Okay I'll change that